tests/test_merge_calendars_with_date_recurrence_id.py

```diff
@@ -0,0 +1,23 @@
+from datetime import date, datetime
+from typing import Any
+
+
+def test_merge_calendars_with_date_recurrence_id(merge: Any) -> None:
+    """Test that merging calendars works with both date and datetime recurrence IDs."""
+    merged = merge(["calendar_with_date_recurrence_id.ics"])
+    events = list(merged.walk("VEVENT"))
```

Comment:
with the latest icalendar, you can do merged.events

Reply:
Good to know